drapps/README.md

```diff
@@ -0,0 +1,102 @@
+# DRApps
+
+## About 
+
+DRApps is simple CLI tool for creating your custom application in Data Robot cloud.
+
+Custom application it small web server hosted by Data Robot in Kubernetes environment.
+Data Robot do not provide any storage resources for this server, but you can use Data
+Robot API and other Internet services.
```

Comment:
The user is probably not concerned whether we run their app in kubernetes or not. That's an implementation detail we don't need to tell them about.

```suggestion
Custom applications does not provide any storage but you may access the full DataRobot API and other Internet services
```